snuba/datasets/configuration/generic_metrics/storages/distributions.yaml

```diff
@@ -0,0 +1,85 @@
+version: v1
+kind: readonly_storage
+name: generic-metrics-distributions
+
+storage:
+  key: generic_metrics_distributions
+  set_key: generic_metrics_distributions
+
+schema:
+  columns:
+    [
+      { name: org_id, type: UInt, args: { size: 64 } },
+      { name: use_case_id, type: String },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: metric_id, type: UInt, args: { size: 64 } },
+      { name: timestamp, type: DateTime },
+      { name: retention_days, type: UInt, args: { size: 16 } },
+      {
+        name: tags,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: key, type: UInt, args: { size: 64 } },
+                { name: indexed_value, type: UInt, args: { size: 64 } },
+                { name: raw_value, type: String },
+              ],
+          },
+      },
+
+      {
+        name: _raw_tags_hash,
+        type: Array,
+        args: { type: UInt, arg: 64, schema_modifiers: [readonly] },
+      },
+      {
+        name: _indexed_tags_hash,
+        type: Array,
+        args: { type: UInt, arg: 64, schema_modifiers: [readonly] },
+      },
+      { name: granularity, type: UInt, args: { size: 8 } },
+
+      {
+        name: percentiles,
+        type: AggregateFunction,
+        args:
+          {
+            func: "quantiles(0.5, 0.75, 0.9, 0.95, 0.99)",
+            arg_types: [{ type: Float, arg: 64 }],
+          },
+      },
+      {
+        name: min,
+        type: AggregateFunction,
+        args: { func: "min", arg_types: [{ type: Float, arg: 64 }] },
+      },
+      {
+        name: max,
+        type: AggregateFunction,
+        args: { func: "max", arg_types: [{ type: Float, arg: 64 }] },
+      },
+      {
+        name: avg,
+        type: AggregateFunction,
+        args: { func: "avg", arg_types: [{ type: Float, arg: 64 }] },
+      },
+      {
+        name: sum,
+        type: AggregateFunction,
+        args: { func: "sum", arg_types: [{ type: Float, arg: 64 }] },
+      },
+      {
+        name: count,
+        type: AggregateFunction,
+        args: { func: "count", arg_types: [{ type: Float, arg: 64 }] },
+      },
+      {
+        name: histogram_buckets,
+        type: AggregateFunction,
+        args: { func: "histogram(250)", arg_types: [{ type: Float, arg: 64 }] },
+      },
+    ]
+
+query_processors: ["TableRateLimit", "TupleUnaliaser"]
```

Comment:
Do we plan to have a list of query processors that are added by default? I think there are a few that do not need to be specified. 
Example `TableRateLimit`. A rate limiter per table should always be present. There should be no need to ever remove it.

Reply:
I've been 1-1 mapping existing logic to config so far, we can think about or introduce this always-add logic once we get the config stuff working (I'll create a tech debt task so it's not forgotten)